nixos/tests/web-servers/stargazer.nix

```
@@ -1,4 +1,47 @@
 { pkgs, lib, ... }:
+let
+  test_src = pkgs.stargazer.src;
+  test_script = pkgs.stdenv.mkDerivation rec {
+    pname = "stargazer-test-script";
+    version = pkgs.stargazer.version;
+    src = test_src;
+    buildInputs = with pkgs; [ (python3.withPackages (ps: with ps; [ cryptography ])) ];
+    dontBuild = true;
+    dontConfigure = true;
+    doCheck = false;
+    installPhase = ''
+      mkdir -p $out/bin
+      cp scripts/gemini-diagnostics $out/bin/test
+    '';
+  };
+  test_env = pkgs.stdenv.mkDerivation rec {
+    pname = "stargazer-test-env";
+    version = pkgs.stargazer.version;
+    src = test_src;
+    configurePhase = "true";
```

```suggestion
    dontConfigure = true;
```
but is that truly required? If the configure file is missing, the step is just skipped